.circleci/config.yml

```diff
@@ -2,100 +2,20 @@ version: 2
 
 source_dependency_versions: &source_dependency_versions
   MSGPACK_VERSION: 3.0.1
-  OPENTRACING_VERSION: 1.5.0
-  NGINX_OPENTRACING_VERSION: 0.5.0
-
-image_ubuntu_18_04: &image_ubuntu_18_04
-  docker:
-    - image: datadog/docker-library:dd_opentracing_cpp_ubuntu_18.04_0_2_1
-
-image_ubuntu_16_04: &image_ubuntu_16_04
-  docker:
-    - image: datadog/docker-library:dd_opentracing_cpp_ubuntu_16.04_0_2_1
-
+  OPENTRACING_VERSION: 1.4.0
+  CURL_VERSION: 7.60.0
 
 defaults: &defaults
   working_directory: ~/dd-opentracing-cpp
+  docker:
+    - image: datadog/docker-library:dd_opentracing_cpp_ubuntu_18.04_0_2_1
```

Comment:
None that need to be made, but I can remove ruby, gem and fpm! I'll do that sooner rather than later. And later; make it an alpine image for great bandwidth savings!